server/src/main/java/org/elasticsearch/index/engine/Engine.java

```diff
@@ -698,11 +712,6 @@ public final CommitStats commitStats() {
      */
     public abstract long getLastSyncedGlobalCheckpoint();
 
-    /**
-     * Returns the latest local checkpoint value that has been processed
-     */
-    public abstract long getProcessedLocalCheckpoint();
```

Comment:
As far as I can tell this method was never present in ES in `Engine`, only in `InternalEngine`.